extensions/sftp/processors/PutSFTP.cpp

```diff
@@ -184,15 +184,15 @@ void PutSFTP::onSchedule(const std::shared_ptr<core::ProcessContext> &context, c
   }
   context->getProperty(ConflictResolution.getName(), conflict_resolution_);
   if (context->getProperty(RejectZeroByte.getName(), value)) {
-    utils::StringUtils::StringToBool(value, reject_zero_byte_);
+    reject_zero_byte_ = utils::StringUtils::toBool(value).value_or(false);
   }
   if (context->getProperty(DotRename.getName(), value)) {
-    utils::StringUtils::StringToBool(value, dot_rename_);
+    dot_rename_ = utils::StringUtils::toBool(value).value_or(false);
```

Comment:
Shouldn't this default to true?

Reply:
In the initialiser, it is set to false therefore I went with that. Why do you think the default should be true?

Reply:
Fixed it after stand-up.